### ISSUE TITLE
License

### ISSUE THREAD

**Issue by Hamath-Eltrael** [abdullahkhursheed22/factsilove#3]
I'd recommend to add a license. Just to avoid any problems that might arise in the future. If you want to take the easy route you can get a CC(Creative Commons) License.

If it were my project I'd probably go for the [CC0 v1.0](https://creativecommons.org/publicdomain/zero/1.0/deed.de)

But seeing as it is a more or less legal document I'll refrain from adding something like this as a Pull request. Best of luck to you tough with it.
Or you could just not do it. Also an option! Then it'll default the the law texts.

 I just wanted to inform you of it.

**Comment by Hamath-Eltrael** [comment]
If you want to add an License, I'd recommend adding it to the readme file from #2 and maybe even adding it as a dedicated file on top of that.

If not you can just remove the last couple of line of the readme, or tell me to do it.

**Comment by abdullahkhursheed22** [comment]
Sure! I'll look into the CC license. Thanks again.

**Comment by Hamath-Eltrael** [comment]
Absolutely no problem!  I'm contributing all these QoL changes so I can better contribute other stuff too when I find the time!

**Comment by abdullahkhursheed22** [comment]
Hey! So I applied for a CC license that applies that allows all non-commercial adaptations:

<a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/"><img alt="Creative Commons License" style="border-width:0" src="https://i.creativecommons.org/l/by-nc/4.0/88x31.png" /></a><br />This work is licensed under a <a rel="license" href="http://creativecommons.org/licenses/by-nc/4.0/">Creative Commons Attribution-NonCommercial 4.0 International License</a>.

**Comment by abdullahkhursheed22** [comment]
It'd be helpful if you could let me know if it's the right one or if it requires any changes. Thanks!

**Comment by Hamath-Eltrael** [comment]
Ok, as far as I'm aware it's the one Wikipedia is using! I'll look into it! Might take me a day or two though, and it will be in no way, shape or form legal advice. And I want to let you know that I have no legal experience or knowledge and can only use my common sense as far as possible.

**Comment by Hamath-Eltrael** [comment]
Oh and I'd love to know what you have used as sources so far! Might be important for Copy-Paste text(if it's in the project) and Pictures(and other media). If you don't know anymore, that's ok. Then I'll try to find your sources. But It'd be helpful to know how the Texts came into existence. 

If you just read one or more texts and then wrote the text that is used in the project, we can just keep it as is, no sources needed(it's not a school project or something along the line). But if it's copied we have to credit the author (or rewrite) according to their License. So let me know what it is.

For the Media I'll probably have to go to google and find the Sources that way. Time Consuming but doable.

**Comment by abdullahkhursheed22** [comment]
> and it will be in no way, shape or form legal advice.
I understand, of course. I have no issue if someone wants to use the content as I learned all concepts from other resources in the first place.

> Oh and I'd love to know what you have used as sources so far! 
All the text is original i.e., nothing has been copy-pasted. I jotted down the information from varying sources, including lectures, podcasts, books, and YouTube videos. So the content has been rephrased.

> For the Media I'll probably have to go to google and find the Sources that way. Time Consuming but doable.
The media is either screenshots from videos or images from Google. I think finding the sources would be too much of a hassle. If it wouldn't be a major problem, we can probably skip this process.

Thanks again for your help!
